scripts/AlternateImageVersion.ps1

```diff
@@ -1,44 +1,28 @@
 param (
-    [Parameter(Mandatory=$false)]
+    [Parameter(Mandatory=$true)]
     [string]
     $Gatestack,
-
     [Parameter(Mandatory=$true)]
     [string]
     $Gateversion,
-
     [Parameter(Mandatory=$false)]
     [string]
-    $Region,
-
-    [Parameter(Mandatory=$false)]
-    [string]
-    $childstack
+    $Region
 )
-
 if ( $Region -ne "" ) {
     Write-Host ("Warning: Region $Region is ignored")
 }
 
-if (($Gatestack -eq "" -and $Childstack -eq "") -or ($Gatestack -ne "" -and $Childstack -ne "")) {
-    Write-Host ("Either -Gatestack or -Childstack MUST be specified, but not both")
-    throw
-}
-
-$SkuName = "$($Gateversion)"
+Write-Host "Gateversion1: $Gateversion Gatestack1: $Gatestack Gateversion2: $($Gateversion) Gatestack2: $($Gatestack)"
```

Comment:
Tidy this up so you are just echoing the parameters passed in to the function. Remove the numbers.